huxley/api/tests/test_delegate.py

```diff
@@ -597,4 +597,4 @@ def test_superuser_fail(self):
             {'id': self.delegate2.id, 'assignment': self.new_assignment.id}
         ]
 
-        self.assertRaises(ValidationError, self.get_response, self.school.id)
+        self.assertRaises(ValidationError, self.get_response, self.school.id)
```

Comment:
revert, and/or configure your editor to automatically add a newline at the end of files.